flow/activities/flowable.go

```diff
@@ -336,9 +338,10 @@ func (a *FlowableActivity) SyncFlow(
 	normRequests := make(chan NormalizeBatchRequest, normalizeBufferSize)
 
 	group, groupCtx := errgroup.WithContext(ctx)
+	normalizeCtx := internal.WithOperationContext(groupCtx, protos.FlowOperation_FLOW_OPERATION_NORMALIZE)
```

Comment:
can move this into the group.Go scope

Reply:
groupCtx also involves `syncRecords` and `maintainReplConn` which I think needs to be in the `sync` operation?